### ISSUE TITLE
Looks like igraph example is failing in the CI

### ISSUE THREAD

**Issue by dschult** [networkx/networkx#4894]
The most recent PRs have a build error for the examples with the following traceback.
Perhaps the API has changed due to a recent release?
```
Sphinx-Gallery successfully executed 60 out of 61 files subselected by:

    gallery_conf["filename_pattern"] = '/plot'
    gallery_conf["ignore_pattern"]   = '__init__\\.py'

after excluding 0 files that had previously been run (based on MD5).


Extension error:
Here is a summary of the problems encountered when running the examples

Unexpected failing examples:
/home/circleci/repo/examples/external/plot_igraph.py failed leaving traceback:
Traceback (most recent call last):
  File "/home/circleci/repo/examples/external/plot_igraph.py", line 39, in <module>
    ig.plot(h, layout=layout, target=ax1)
  File "/home/circleci/repo/venv/lib/python3.8/site-packages/igraph/drawing/__init__.py", line 480, in plot
    result.draw(obj, *args, **kwds)
  File "/home/circleci/repo/venv/lib/python3.8/site-packages/igraph/drawing/graph.py", line 1114, in draw
    for idx, color in kwds["mark_groups"].items():
AttributeError: 'bool' object has no attribute 'items'
```

**Comment by dschult** [comment]
They have a new release 0.9.5 on June 11 and the changelog says they have a new input argument `mark_groups` that now works. But if you don't specifying it, it doesn't work.

**Comment by dschult** [comment]
I've opened [an Issue](https://github.com/igraph/python-igraph/issues/415) at `python-igraph` which is where the error occurs.

**Comment by iosonofabio** [comment]
My bad, I was changing some infra code for plotting communities and this one slipped through the cracks. I think we'll have a bugfix release for this, and then we should try and implement some plotting CI like you guys or mpl have.

**Comment by dschult** [comment]
I can confirm that the fix works for our example/use-case.

FYI @iosonofabio -- our plotting CI doesn't test that the graph made is
correct (not yet -- we're working on that). It just makes sure it runs successfully.
That might be an easier CI smoke test that a full blown plotting CI.

**Comment by dschult** [comment]
OK... This is fixed in igraph 0.9.6 which has been released.  The tests in #4890 are now passing.
Thanks @iosonofabio
